dist/docker/debian/build_docker.sh

```diff
@@ -21,19 +21,23 @@
 # along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-product="$(<build/SCYLLA-PRODUCT-FILE)"
-version="$(<build/SCYLLA-VERSION-FILE)"
-release="$(<build/SCYLLA-RELEASE-FILE)"
-
-mode="release"
-
-if uname -m | grep x86_64 ; then
-  arch="amd64"
-fi
-
-if uname -m | grep aarch64 ; then
-  arch="arm64"
-fi
+MODE="release"
```

Comment:
Please avoid the refactoring, it makes it impossible to read the patch. Also, lower-case variables are perfectly fine for bash.